gems/arabic-prawn/CVE-2014-2322.yml

```diff
@@ -0,0 +1,24 @@
+---
+gem: arabic-prawn
+cve: 2014-2322
+ghsa: hgmw-x865-hf9x
+url: http://www.openwall.com/lists/oss-security/2014/03/10/8
+title: Arabic Prawn allows remote attackers to execute arbitrary commands via shell
+  metacharacters
+date: 2017-10-24
+description: |
+  "lib/string_utf_support.rb" in the Arabic Prawn 0.0.1 gem for Ruby allows
+  remote attackers to execute arbitrary commands via shell metacharacters in the
+  (1) downloaded_file or (2) url variable.
+patched_versions:
+  - "> 0.0.1"
+related:
+  url:
+    - https://nvd.nist.gov/vuln/detail/CVE-2014-2322
+    - http://www.openwall.com/lists/oss-security/2014/03/10/8
+    - http://www.openwall.com/lists/oss-security/2014/03/12/6
+    - https://web.archive.org/web/20160306235714/http://www.vapid.dhs.org/advisories/arabic-ruby-gem.html
+    - http://www.vapid.dhs.org/advisories/arabic-ruby-gem.html
+    - https://github.com/advisories/GHSA-hgmw-x865-hf9x
+    - https://rubygems.org/gems/Arabic-Prawn
+notes: "Never patched"
```

Comment:
I can't find [arabic-prawn](https://rubygems.org/gems/arabic-prawn) on rubygems.org, but there are [prawn-arabic 0.0.6](https://rubygems.org/gems/prawn-arabic) and [Arabic-Prawn 0.0.1](https://rubygems.org/gems/Arabic-Prawn). Was this patched, but under a different gem name?

Reply:
FYI: gems/Arabic-Prawn/CVE-2014-2322.yml is already in the database so:
I can merged the new data into existing file, deleted both files, or delete just my file.

Reply:
Yeah merging into `gems/Arabic-Prawn/CVE-2014-2322.yml` sounds better.